esp/esp/utils/widgets.py

```diff
@@ -219,6 +219,8 @@ def render(self, name, value, attrs=None, choices=()):
         return mark_safe(output)
 
 class NavStructureWidget(forms.Widget):
+    # TODO(benkraft): Convert this to an actual static script so we don't have
```

Comment:
Yes, will double-check but I think fruitsalad is the only other one using this widget and I tested that.